src/api/v1alpha1/inspectionpolicy_types.go

```diff
@@ -60,13 +60,15 @@ type DataProvider struct {
 	// +kubebuilder:validation:Optional
 	Cache *Cache `json:"cache,omitempty"`
 	// Connection settings.
+	// +kubebuilder:validation:Optional
```

Comment:
this change is not required as we are not using as part of governor.

Reply:
Changes are done for complete CRD, not only for the governor.
Any field which has default has to be optional. Otherwise, there is no point in having a default value.

Reply:
DataProvider is an unused structure. I am reverting these changes as there is no impact